pkgs/api/Makefile

```diff
@@ -8,9 +8,9 @@ all: format codegen lint test
 
 .PHONY:codegen
 codegen:
-	elegen folder -d specs -o codegen -g openapi3 || exit 1
+	/Users/knainwal/apomux/workspace/code/go/src/go.aporeto.io/elemental/cmd/elegen/elegen folder -d specs -o codegen -g openapi3 || exit 1
```

Comment:
these shouldn't be committed

Reply:
Oops. Thanks for spotting this Sawyer. Accidentally left this over.